src/components/dialog/Dialog.spec.js

```diff
@@ -69,6 +69,38 @@ describe('BDialog', () => {
         expect(wrapper.vm.onConfirm).toHaveBeenCalled()
     })
 
+    it('async confirm and keep Loading state', async () => {
+        wrapper.setProps({
+            onConfirm: jest.fn((confirmValue, { startLoading }) => {
+                startLoading()
+                expect(wrapper.vm.isLoading).toBeTruthy()
+                return new Promise(() => {})
+            }),
+            closeOnConfirm: false
+        })
+        expect(wrapper.vm.isLoading).toBeFalsy()
+        wrapper.vm.confirm()
+        expect(wrapper.vm.isLoading).toBeTruthy()
+        expect(wrapper.vm.onConfirm).toHaveBeenCalled()
+    })
+
+    it('async confirm and close Loading state', async () => {
+        wrapper.setProps({
+            onConfirm: jest.fn((confirmValue, { startLoading, cancelLoading }) => {
+                startLoading()
+                expect(wrapper.vm.isLoading).toBeTruthy()
+                return new Promise(() => {
+                    cancelLoading()
+                })
```

Comment:
Above three lines are almost equivalent to just calling `cancelLoading()`.

Reply:
It is OK since it works, though, could you replace the three lines with a simple line:
```
cancelLoading()
```

Reply:
@kikuomax 

Done

Is this what you wanted?

Reply:
Perfect!